knowledge-base/grid-inline-edit-form.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Add inline Telerik Form to the the Grid Rows
+description: How to add inline Telerik Form to the Grid Rows.
+type: how-to
+page_title: Add inline Telerik Form to the the Grid Rows
+slug: grid-kb-inline-form
+position: 
+tags: telerik,blazor,grid,inline,form,rows
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I want to add an inline Telerik Form to the Grid rows when I enter edit mode.
+
+
+## Solution
+
+Add an inline [Telerik Form]({%slug form-overview%}) to the Grid rows by following these steps:
+
+* Define a [DetailTemplate]({%slug components/grid/features/hierarchy%}).
```

Comment:
This seems like a step-by-step procedure. In this case, the recommendation is to go with a numbered list. 

Numbered lists represent procedures where the order of the steps is important.
Unordered lists work for procedures where the order of completing the steps isn't important. 

You can read more about lists [here](https://docs.telerik.com/style-guide/lists).